python/interpret_community/widget/js/package.json

```diff
@@ -19,7 +19,7 @@
       "babel-preset-es2015": "6.24.1",
       "babel-preset-react": "6.24.1",
       "css-loader": "2.1.1",
-      "interpret-dashboard": "0.1.8",
+      "interpret-dashboard": "0.1.7-beta6",
```

Comment:
why is this version decremented down?  maybe it should be more like 0.1.8-beta6? or 0.1.9 at this point?

Reply:
the 7-beta is the v2 dashboard, 8 and 9 are the patched v1 dashboard, once this PR is settled, I can push a 0.1.10 or 0.1.10-beta1 that has both the latest v1 and the new v2 dashboard together. The widget can pull that in.